src/main/java/com/contrastsecurity/rO0/RO0Agent.java

```diff
@@ -20,22 +34,81 @@ public static void agentmain(String args, Instrumentation inst) throws IOExcepti
 	private static void setup(String args, Instrumentation inst) {
 		ClassFileTransformer xform = new RO0Transformer();
 		inst.addTransformer(xform);
+		readEnvironment();
+	}
+	
+	private static void readEnvironment() {
+		
+		String outfile = System.getProperty("RO0.outfile");
```

Comment:
Would love for our System properties to be "RO0" -> "rO0". I only made it with a capital R when I had to -- when forced by Java's rules about class names.